src/modules/navigator/mission.cpp

```diff
@@ -440,7 +444,21 @@ Mission::set_mission_items()
 		}
 
 		/* if we just did a takeoff navigate to the actual waypoint now */
-		if (_work_item_type == WORK_ITEM_TYPE_TAKEOFF) {
+		if (_work_item_type == WORK_ITEM_TYPE_TAKEOFF) {			
+
+			/* handle VTOL TAKEOFF command */
+			if(_takeoff_vtol_transition){
```

Comment:
I have an interesting question here. Do you want to navigate to the takeoff waypoint before or after the transition? ;)
And another one: is it really the idea of this waypoint that you transition to FW after? I can see why it makes sense, but it's called VTOL takeoff, we might be misinterpreting that.